lms/djangoapps/student_account/test/test_views.py

```diff
@@ -502,12 +504,29 @@ def test_context(self):
 
     def test_view(self):
 
-        view_path = reverse('account_settings')
-        response = self.client.get(path=view_path)
+        response = self.client.get(path=self.view_path)
 
         for attribute in self.FIELDS:
             self.assertIn(attribute, response.content)
 
+    def test_header_with_programs_listing_enabled(self):
+        """
+        Verify that tabs header will be shown while program listing is enabled.
+        """
+        self.create_programs_config(program_listing_enabled=True)
+        response = self.client.get(path=self.view_path)
+
+        self.assertContains(response, "Programs")
```

Comment:
If the word "Programs" shows on the dashboard page as part of a course, this test would become useless. Can we be more specific on checking the existence of the header navigation? Say the existence of a CSS class?